docs/tutorials/pre_executed/map_partitions.ipynb

```diff
@@ -54,24 +54,18 @@
    "cell_type": "code",
```

Comment:
<p>lets use <code>Open a catalog</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/792/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I like it. Will fix.</p>